client/src/components/HeaderFilterItems/HeaderFilterItems.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {Box, Paper, MenuList, MenuItem,Typography} from '@mui/material';
+import { styled } from '@mui/system';
+
+const items = [
+    {
+        image: '../../assets/images/headerFilter/1.jpeg',
+        name: 'karpaty - 1 day + spa',
+        price: ' 4000 uah',
+        amountOfDay: '1 day',
+    },
+    {
+        image: '../../assets/images/headerFilter/2.jpeg',
+        name: 'winter karpaty',
+        price: ' 3500 uah',
+        amountOfDay: '2 day',
+    },
+    {
+        image: '../../assets/images/headerFilter/3.jpeg',
+        name: 'weel in karpaty',
+        price: ' 7000 uah',
+        amountOfDay: '7 day',
+    },
+  ];
+
+const ItemBox = styled((props) => <Box  {...props} />)(() => ({
+    paddingLeft: 0,
+    display: 'grid',
+    gridTemplateColumns: 'auto auto',
+    gridTemplateRows: 'repeat(2, 1fr)',
+    gridColumnGap: ' 10px',
+    gridRowGap: '0px',
+    boxSizing: 'border-box'
+}));
+
+const PaperContainer = styled((props) => <Paper  {...props} />)(({ theme }) => ({
```

Comment:
done

Reply:
@valeriaholdovych  no its not done.. i dont understand why you comment dont while it is not.

Reply:
@Ahmed-Fahmy-Dev don't know wy my latest updates not uploaded. I'll fix it in the next commit

Reply:
@Ahmed-Fahmy-Dev by the way I cant remove props from my created container, because in this way it doesn't work correctly. I need tis props. but, I can write properties in sx in <Paper/> component